src/test/java/org/opentripplanner/GtfsTest.java

```diff
@@ -56,7 +56,7 @@ protected void setUp() {
         File gtfs = new File("src/test/resources/" + getFeedName());
         File gtfsRealTime = new File("src/test/resources/" + getFeedName() + ".pb");
         GtfsBundle gtfsBundle = new GtfsBundle(gtfs);
-        feedId = new GtfsFeedId.Builder().id("FEED").build();
+        feedId = new GtfsFeedId.Builder().id("").build();
```

Comment:
I have no context for this change. If we're needing to make this change, does that mean it's breaking?

Reply:
The changes made are sort of breaking in that an error will now be thrown if OTP reads in two feeds that have the same feed_id. This method is used primarily for testing purposes so it's unlikely that anyone would use this in their own code.